src/app/dao/PartDAO.ts

```diff
@@ -19,4 +19,14 @@ export class PartDAO extends FirebaseFirestoreDAO<IPart> {
     public observeActiveParts(callback: FirebaseCollectionObserver<IPart>): () => void {
         return this.observingWhere([['result', '==', MGPResult.UNACHIEVED.value]], callback);
     }
+    public async userHasActivePart(username: string): Promise<boolean> {
+        // This can be simplified into a simple query once part.playerZero and part.playerOne are in an array
```

Comment:
looks like a todo to me !

Reply:
un peu, cela nécéssite un refactor des Part pour le faire en une seule requête cependant ! Je tickette ?

Reply:
Oui s'il te plai !
Y a un ticket super vieux sur 'optimiser les DAO' je crois, peut être l'unique ticket étiquetté DAO :p

Reply:
ticketté